ingestion/src/metadata/ingestion/bulksink/metadata_usage.py

```diff
@@ -251,11 +274,16 @@ def __get_table_joins(
             column_joins_dict[column_join.tableColumn.column] = joined_with
 
         for key, value in column_joins_dict.items():
-            key_name = get_column_fqn(table_entity=table_entity, column=key).split(".")[
-                -1
-            ]
+            key_name = get_column_fqn(table_entity=table_entity, column=key)
+            if not key_name:
+                logger.warning(
+                    f"Could not find column {key} in table {table_entity.fullyQualifiedName.__root__}"
+                )
+                continue
             table_joins.columnJoins.append(
-                ColumnJoins(columnName=key_name, joinedWith=list(value.values()))
+                ColumnJoins(
+                    columnName=key_name.split(".")[-1], joinedWith=list(value.values())
```

Comment:
Should we use `fqn.split` method here to handle this better? 🤔

Reply:
that's a good point, copy pasting from 0.10.3 was not the best idea :P I'll fix that